src/NServiceBus.Core.Tests/Pipeline/MutateTransportMessage/MutateOutgoingTransportMessageBehaviorTests.cs

```diff
@@ -24,7 +24,7 @@ public void Should_throw_friendly_exception_when_IMutateOutgoingTransportMessage
         [Test]
         public async Task When_no_mutator_updates_the_body_should_not_update_the_body()
         {
-            var behavior = new MutateOutgoingTransportMessageBehavior();
+            var behavior = new MutateOutgoingTransportMessageBehavior(true);
```

Comment:
would it make sense from a readability perspective to use the named params approach here? Of course this would apply to all the tests